### ISSUE TITLE
suggest to support checklisbank

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/nomer#117]
via @myrmoteras

---

ChecklistBank:

Checklist from Plazi are “treatment dataset” we submit to GBIF as DWCA of the entire article and then are repurposed and submitted internally to ChecklistBank where they wait to be incorporated into Catalogue of Life.

 

This works right now only by human curation, including tools to do so, where you can become an editor of parts. Some parts are taken over by the classical data providers such as WORMS who create a whole section.

In works is a semiautomatic tool that will import checklists or part of which do not have any conflicts, eg new species.

 

If you go on Checklistbank  https://www.checklistbank.org/ you can see new checklists in the lower right part,   “latest dataset added” and see a mixture of classical data set and those derived form publications.

 

from https://github.com/plazi/treatments-xml: this is used for creating the RDF for Synospecies based on the Plazi XML. The entities there are treatments.

**Comment by jhpoelen** [comment]
from what I can tell, https://checklistbank.org is a rebranding of Catalogue of Life see attached screenshot take 2022-10-18 .

@myrmoteras @n8upham can you confirm?

![image](https://user-images.githubusercontent.com/1084872/196512979-d4f3d62a-2ac8-433e-827c-42fa5378837e.png)

**Comment by myrmoteras** [comment]
No. It's different. It is the staging area to get checklists into col

New species of the day<https://tb.plazi.org/GgServer/static/newToday.html>
________________________________

**Comment by jhpoelen** [comment]
@myrmoteras thanks for clarifying!
